File: arcade/intro/1 - Intro/2 - century-from-year/main.go
func centuryFromYear(year int) int {
    yearStr := strconv.Itoa(year)
	for len(yearStr) < 4 {
		yearStr = "0" + yearStr
	}
	fmt.Println(yearStr[:2], yearStr[2:])
	f, _ := strconv.Atoi(yearStr[:2])
	s, _ := strconv.Atoi(yearStr[2:])
	if s > 0 {
		return f + 1
	}
	return f
}

File: arcade/intro/1 - Intro/2 - century-from-year/main.rs
fn century_from_year(year: i32) -> i32 {
    if year % 100 == 0 {
        return year / 100;
    }
    (year / 100) + 1
}

fn main() {
    println!("2004 is in the {}st century", century_from_year(2004));
}